site/content/contribute/getting-started/labels.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Labels"
+date: 2018-03-06T00:00:00-04:00
+weight: 5
+subsection: Getting Started
+---
+
+We leverage [GitHub labels](https://help.github.com/en/articles/about-labels) to track the details and lifecycle of issues and pull requests.
+
+# Issue Labels
+* `Area/<name>`: Involves changes to the named area (APIv4, Add E2E Tests, Plugins, etc.)
+* `Difficulty/1:easy`: Easy ticket.
+* `Difficulty/2:medium`: Medium ticket.
+* `Difficulty/3:hard`: Hard ticket.
+* `First Good Issue`: Suitable for first-time contributors.
+* `Framework/<name>`: Requires using the named framework (ReactJS, Redux, etc.)
+* `Help Wanted`: Community help wanted.
+* `Language/<name>`: Requires writing in the named language (Go, JavaScript, etc.)
+* `Localization`: Relates to translating the user interface.
```

Comment:
I propose rename this to `Area/Localization`. This is one of the very old and rarely used labels.

Reply:
Sounds good -- I've listed it as an example in the existing `Area/<name>`.